crypto/types/tx_signature_cache.go

```diff
@@ -0,0 +1,98 @@
+package types
+
+import (
+	"sync/atomic"
+
+	"github.com/VictoriaMetrics/fastcache"
```

Comment:
Why introduce an additional cache dependency into the SDK when we already use Hashi's LRU cache. I can't see why an LRU wouldn't serve our needs here too.